Middleware/EventManagerMiddleware.cs

```diff
@@ -39,43 +39,44 @@ IOptions<EventManagerConfiguration> config
                 {
                     ExternalServiceConfiguration externalService = _config.ExternalServices.Find(x => x.Name == eventSubscriberConf.Name);
 
-                    if (externalService != null)
+                    if (externalService == null)
```

Comment:
The only thing that changed here is that I refactored this `if` so that it would continue if external service is `null`. Nothing else was changed